src/main/webapp/app/shared/pipes/artemis-date.pipe.ts

```diff
@@ -135,10 +135,10 @@ export class ArtemisDatePipe implements PipeTransform, OnDestroy {
         if (!long) {
             switch (locale) {
                 case 'de':
-                    format = 'DD.MM.YYYY';
+                    format = 'DD.MM.YY';
                     break;
                 default:
-                    format = 'YYYY-MM-DD';
+                    format = 'YY-MM-DD';
```

Comment:
Students might now get confused since it is no longer clear that the year is the first value here. E.g. the date 24.01.23/23-01-24 can lead to problems when only briefly looking at the supposed day 23. 
There has to be some kind of measurement against this confusion.

Reply:
I think there is no Date format in which the year, month, and day order is clear, which is shorter than "YYYY-MM-DD" and "DD.MM.YYYY".

But then we have again the case, that the "short-date" is not noticeably shorter than the "long-date". which would make the window width dependent change of format unnecessary, since the short date is not that much shorter to give more space on mobile, wouldn't it?

https://github.com/ls1intum/Artemis/blob/ef494745a6f67e481187d202d04aeb1273512a3b/src/main/webapp/app/shared/pipes/artemis-date.pipe.ts#L158-L162

What do you think about changing the short-format back to the "long-year" format, but removing the window width dependence?
(my other suggestion would be to extend the long-date format with the long-month format - September instead of Sept - and keep the window dependence then, but it would be an unnecessary increase of length then)

Reply:
I would remove the window width dependence for this particular case

Reply:
Thank you for your feedback! I hope I got your suggestions correctly and I apologize for the long discussion.

- I readded the "YYYY" to the short-date to avoid confusion among users, which number is the year value
- The short-date format is used in this particular case independent from the window width.

Please have another look at the PR